temporal-sdk/src/main/java/io/temporal/client/WorkflowClient.java

```diff
@@ -568,6 +568,302 @@ static <A1, A2, A3, A4, A5, A6, R> WorkflowExecution start(
     return WorkflowClientInternalImpl.start(workflow, arg1, arg2, arg3, arg4, arg5, arg6);
   }
 
+  /**
+   * Executes zero argument workflow with void return type together with an additional operation.
+   *
+   * @param workflow The only supported value is method reference to a proxy created through {@link
+   *     #newWorkflowStub(Class, WorkflowOptions)}.
+   * @param additionalOperation to run at the beginning of the workflow (e.g. {@link
+   *     WorkflowStartOperationUpdate} for Update-With-Start)
+   * @return WorkflowExecution that contains WorkflowId and RunId of the started workflow.
```

Comment:
The returns are wrong now no?

Reply:
👍 let me fix that.

Reply:
addressed here https://github.com/temporalio/sdk-java/pull/2199/commits/cb7565900ff4d33696fb36025723dd523a3e2927